VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -25,51 +25,20 @@
 import org.edx.mobile.util.Config;
 import org.edx.mobile.util.ResourceUtil;
 
+import java.util.ArrayList;
 import java.util.Date;
-import java.util.HashMap;
-import java.util.regex.Pattern;
+import java.util.List;
 
 public abstract class DiscussionTextUtils {
-    private static final Pattern PATTERN_DUPLICATE_WHITESPACE = Pattern.compile("\\s+");
-
     @Inject
     private static Config config;
 
+    private static final char WHITESPACE = ' ';
```

Comment:
I don't think there's much point in declaring a constant for the whitespace character; we can use the literal directly where needed.